src/screens/FeedScreen/RenderList.js

```diff
@@ -47,7 +53,9 @@ const RenderListFeed = (props) => {
     source = SOURCE_FEED_TAB,
     hideThreeDot = true,
     showAnonymousOption = false,
-    onHeaderOptionClicked = () => {}
+    onDeletePost,
+    isShowDelete,
+    isSelf
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The props `onDeletePost`, `isShowDelete`, and `isSelf` have been added. Ensure that these props are passed from the parent component and handled correctly.
```diff
+ onDeletePost,
+ isShowDelete,
+ isSelf
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->